### ISSUE TITLE
How do I use this?

### ISSUE THREAD

**Issue by banesullivan** [rtfd/sphinx-notfound-page#5]
How can I plug this into my existing docs? I `pip` installed this then tried adding it to the `extensions` in my `conf.py`:

```
$ make html
Running Sphinx v1.8.4

Extension error:
Could not import extension sphinx-notfound-page (exception: No module named 'sphinx-notfound-page')
make: *** [html] Error 2
```

**Comment by banesullivan** [comment]
Actually, figured it out by looking for the `setup` function in the source:

Add `'notfound.extension'` to the `extensions` in your `conf.py`

**Comment by humitos** [comment]
Hi! Thanks for posting that you figured it out and the solution.

I'm pending on writing the docs #1 

Feel free to open another issue if you need more help.
